okteto/okteto
# CLI Quality - Lint - gci

# Proposed changes

Fixes https://okteto.atlassian.net/browse/LAKE-68

This PR adds `gci` as enabled linter and fixes de error raised by it. It does not affect production code, just the imports organization on each file

## How to validate

Configuration for gci linter is using the default one, so import packages are now sorted in a determined way.
You can try to move around imports from any file and run `make lint`, this should return error.

In order to fix them i used `--fix` option running `golangci-lint` locally

## CLI Quality Reminders 🔧

For both authors and reviewers:

- Scrutinize for potential regressions
- Ensure key automated tests are in place
- Build the CLI and test using the validation steps
- Assess Developer Experience impact (log messages, performances, etc)
- If too broad, consider breaking into smaller PRs
- Adhere to our [code style](https://github.com/okteto/okteto/blob/master/docs/code-style.md) and [code review](https://github.com/okteto/okteto/blob/master/docs/code-review.md) guidelines
